### PR TITLE
Sign rulesets

### PR DESCRIPTION
**Pull request by Hainish**

### CODE REVIEW

File: chromium/background-scripts/update.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ let store,
   background_callback;
 
 // how often we should check for new rulesets
-const periodicity = 10;
+const periodicity = 7200;
```
Comment (cschanaj) [inline chromium/background-scripts/update.js]: @Hainish isn't this just 2 hours instead of 2 days in seconds?

Comment (Hainish) [reply chromium/background-scripts/update.js]: You're definitely right!  Thank you.

Comment (Hainish) [reply chromium/background-scripts/update.js]: It's those little bugs which are killers.  Glad you caught that.

Comment (cschanaj) [reply chromium/background-scripts/update.js]: 033f82d looks good to me, thanks! 😄

**Pull request merged**